.goreleaser.yaml

```diff
@@ -20,7 +20,7 @@ brews:
   - tap:
       owner: thegreenwebfoundation
       name: homebrew-grid-intensity
-      token: {{ .Env.HOMEBREW_TAP_GITHUB_TOKEN }}
+      token: "{{ .Env.HOMEBREW_TAP_GITHUB_TOKEN }}"
```

Comment:
@mrchrisadams I missed the quotes here 🤦‍♂️ 

Just going to merge this since its a minor change 🙏